nailgun/entities.py

```diff
@@ -2912,6 +2912,38 @@ def __init__(self, server_config=None, **kwargs):
             'to': entity_fields.IPAddressField(null=True),
             'vlanid': entity_fields.StringField(null=True),
         }
+        if (getattr(server_config, 'version', parse_version('6.1')) >=
+            parse_version('6.1')):
```

Comment:
The linters will complain about this line. Can you indent this by another four spaces?